core/dbt/artifacts/resources/v1/model.py

```diff
@@ -51,4 +52,10 @@ def __post_serialize__(self, dct: Dict, context: Optional[Dict] = None):
         dct = super().__post_serialize__(dct, context)
         if context and context.get("artifact") and "defer_relation" in dct:
             del dct["defer_relation"]
+
+        # delete extra keys that should't be serialized
```

Comment:
@gshank @QMalcolm I figured out the reason `batch` suddenly started to show up is because we allow extra on Model, so when we convert ModelNode to Model when writing an artifact, it will be kept as extra and convert into keys in dct.

Reply:
Ohhhhhhhhh that makes sense. Thank you for figuring this out and removing it with the below process 👍